src/app-layer-ssl.c

```diff
@@ -3048,9 +3214,10 @@ void RegisterSSLParsers(void)
         }
         SCLogDebug("ssl_config.encrypt_mode %u", ssl_config.encrypt_mode);
 
+#ifdef HAVE_JA3
+        const char *strval = NULL;
```

Comment:
Maybe put ja3 and ja4 each in its own function guarded by the right ifdef would be more readable ?

Reply:
Can do.